docs/how-to-guides/themes/theme-json.md

```diff
@@ -981,9 +981,9 @@ Currently block variations exist for "header" and "footer" values of the area te
 
 ## Developing with theme.json
 
-It can be difficult to remember the theme.json settings and properties while you develop, so a JSON scheme was created to help. The schema is available at [SchemaStore.org](https://schemastore.org/)
+It can be difficult to remember the theme.json settings and properties while you develop, so a JSON scheme was created to help. The schema is available at https://schemas.wp.org/trunk/theme.json
 
-Editors can pick up the schema and can provide help like tooltips, autocomplete, or schema validation in the editor. To use the schema in Visual Studio Code, add `"$schema": "https://json.schemastore.org/theme-v1.json"` to the beginning of your theme.json file.
```

Comment:
Is there a plan to create a theme-v1.json version too?

Reply:
No. Versioning can be achieved using branch names, for example: `https://schemas.wp.org/wp/5.9/theme.json` will refer to the schema available in WP 5.9

Reply:
Thank you @mkaz, that would be great but that with that URL I am getting a 404. 🤔